azure_media_services/static/js/player.js

```diff
@@ -46,7 +46,16 @@ function AzureMediaServicesBlock(runtime, element) {
           var divContainer = $("<div class='azure-media-player-toggle-button-style fa fa-quote-left' id='toggleTranscript' role='button' aria-live='polite' tabindex='0'><div class='vjs-control-content'><span class='vjs-control-text'>Toggle</span></div></div>");
           $(".amp-controlbaricons-right").append(divContainer);
 
-          $('#toggleTranscript').click(function() {
+          $('#toggleTranscript').on('click Keydown',(function(evt) {
```

Comment:
Not sure if this was me or you but can you change Keydown to all lowercase: ('click Keydown' => 'click keydown').